src/platform_impl/macos/menu.rs

```diff
@@ -0,0 +1,126 @@
+use cocoa::appkit::{
+    NSApp, NSApplication, NSApplicationActivationPolicyRegular, NSEventModifierFlags, NSMenu,
+    NSMenuItem,
+};
+use cocoa::base::{nil, selector};
+use cocoa::foundation::{NSAutoreleasePool, NSProcessInfo, NSString};
+use objc::runtime::{Object, Sel};
+
+struct KeyEquivalent<'a> {
+    key: &'a str,
+    masks: Option<NSEventModifierFlags>,
+}
+
+pub fn initialize() {
+    unsafe {
+        let _pool = NSAutoreleasePool::new(nil);
+
+        let app = NSApp();
+        app.setActivationPolicy_(NSApplicationActivationPolicyRegular);
+
+        let menubar = NSMenu::new(nil).autorelease();
+        let app_menu_item = NSMenuItem::new(nil).autorelease();
+        menubar.addItem_(app_menu_item);
+        app.setMainMenu_(menubar);
+
+        let app_menu = NSMenu::new(nil).autorelease();
+
+        // About menu item
+        let about_item_prefix = NSString::alloc(nil).init_str("About ");
+        let about_item_title = about_item_prefix
+            .stringByAppendingString_(NSProcessInfo::processInfo(nil).processName());
+        let about_item = menu_item(
+            about_item_title,
+            selector("orderFrontStandardAboutPanel:"),
+            None,
+        )
+        .autorelease();
+
+        // Seperator menu item
+        let sep_first = NSMenuItem::separatorItem(nil).autorelease();
+
```

Comment:
We should add the `Services` menu here (can be done by creating a menu item with an empty menu, and registering that as the [`servicesMenu`](https://developer.apple.com/documentation/appkit/nsapplication/1428608-servicesmenu?language=objc))

Reply:
Hm, not sure it make sense if we don't have any services to add?